### DISCUSSION TITLE
Announcing the 2.36.6 Release

### DISCUSSION THREAD

**Discussion by mapkon** [Versent/saml2aws#1011] [Announcements]
Hello folks!

I am pleased to announce the v2.36.6 release. There has been a tonne of changes in this release, which can be seen [here](https://github.com/Versent/saml2aws/compare/v2.36.5...v2.36.6). This release mainly focused on:

- [Refactored AzureAD authentication for AWS IAM federation](https://github.com/Versent/saml2aws/commit/2be5a3b1d09b2437dd5708404c21a8352b93dd73)
- [Fix various edge cases for not (yet) setup MFA](https://github.com/Versent/saml2aws/commit/a37cca7e9f560fc972b4bbf3f975aa7dfacb2108)
- [Fail on ADFS login errors](https://github.com/Versent/saml2aws/commit/15ba2e5e5e81b024fa62cac842611fbdcd13346d)
- [Add support for building Windows MSI](https://github.com/Versent/saml2aws/commit/146fea1b38985237403540631fbc5a5bec2787a5)
- And much more ...

At the time of writing, the latest binaries have not yet made it to the relevant repositories like [homebrew](https://formulae.brew.sh/formula/saml2aws), but should be updated there in a few hours!

I would like to extend special thanks to the contributors, specifically:

- @gliptak 
- @wolfeidau 
- @briantist 
- @christianmeyer 
- @krzysztofdrys 
- @dboitnot 
- @szechyjs 
- @kstkn 
- @resnikb

If I have missed out anyone, it is not intentional. 

Thank you

**Comment by mapkon** [comment]
FYI:
- @realtimerick
- @kstkn 
- @danishnawab

**Reply by realtimericky** [reply]
Thanks! Works as a charm

**Comment by charltona** [comment]
Thanks for all your hard work team, this is great!

@davidobrien1985 @mapkon 
Could we please have the chocolatey packager updated for Windows users?
https://community.chocolatey.org/packages/saml2aws

**Reply by mapkon** [reply]
> Thanks for all your hard work team, this is great!
> 
> @davidobrien1985 @mapkon Could we please have the chocolatey packager updated for Windows users? https://community.chocolatey.org/packages/saml2aws

@charltona 

Please refer to this [discussion](https://github.com/Versent/saml2aws/discussions/987) for a bit more context regarding windows releases.

Maybe @briantist can help!

**Reply by briantist** [reply]
sorry I don't know anything about the chocolately build and don't use that package.
@charltona there are now Windows MSI packages built as part of CI though, you can get the MSI from the release assets: https://github.com/Versent/saml2aws/releases/tag/v2.36.6

(direct MSI link: https://github.com/Versent/saml2aws/releases/download/v2.36.6/saml2aws_2.36.6_windows_amd64.msi)

**Comment by gliptak** [comment]
Appveyor is failing (again)

https://ci.appveyor.com/project/davidobrien1985/saml2aws/builds/46634992

```
go : go: cannot find GOROOT directory: C:\go120
```

https://github.com/Versent/saml2aws/issues/820

**Comment by mikex99** [comment]
Fresh install on Mac with Brew is not downloading the ms-playwright dependencies automatically.  It just fails when it tries to call them.

**Reply by gliptak** [reply]
please see https://github.com/Versent/saml2aws/pull/1006
